documentation/src/main/asciidoc/migration/index.adoc

```diff
@@ -49,7 +49,7 @@ are the same as those of Hibernate Search {hibernateSearchPreviousStableVersionS
 - JDK 11 or later;
 - Lucene 9 for its Lucene backend;
 - Elasticsearch 7.10+ or OpenSearch 1.3+ for its Elasticsearch backend;
-- Hibernate ORM 6.4.x for the Hibernate ORM integration.
+- Hibernate ORM 6.6.x for the Hibernate ORM integration.
```

Comment:
If you change this, please change the sentence above ("requirements are the same").